docs/open-source/commitment.mdx

```diff
@@ -1,16 +1,19 @@
 ---
-title: Ory is committed to Open Source
+title: Ory is committed to open source
 id: commitment
 sidebar_label: Commitment
 ---
 
 Ory is fundamentally rooted in open source, its community dynamics, and development principles. Ory software is licensed under the
-Apache 2.0 license. The founders and developers working at Ory have decades of experience in open source and open knowledge.
+Apache 2.0 license. The founders and developers working at Ory have decades of experience in open source and open knowledge. Ory
+engineers spent thousands of work hours, committed hundred of thousands of changes, and published millions lines of code under
+[permissive open source licenses such as MIT and Apache 2.0](https://github.com/ory). This deep commitment and history in open
+source software development **isn't a marketing ploy**.
 
 To build open source software and help its community and adoption grow, Ory offers a commercial product which uses open source
 software from Ory and others like the Linux foundation.
```

Comment:
I think it helps both ways since as self-hosting user you also benefit from the development that is funded through the commercial product. 

and its basically 

Ory is OSS -> but offers commercial product -> to build open source software and help its adaption grow (its much easier to adopt Ory if you are using Ory Network, but you are still using open source at its core. 

But there are many nuances and many ways to express this, if you have a suggestion I have no problem to change this :)